### PR TITLE
Add satellite virtual_host_guest to deployment and insights reports

### LINKED ISSUE #1980: Investigate and size whether QPC can gather virtual guest's host from Satellite scans
## Feature description:
Determine difficulty and whether satellite API exposes this information to us.

### Details
Gather virtual guest's host from Satellite scans. Currently, QPC only gather's this information from vCenter scans. Satellite uses virt_who to determine this information. QPC should:

- [ ]   Extract this information from satellite
- [ ]   Include this information in insights reporting

___

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #1980

### CODE REVIEW

File: quipucords/fingerprinter/task.py [hunk 1]
```diff
@@ -1371,7 +1371,7 @@ def _process_vcenter_fact(self, source, fact):
         self._add_fact_to_fingerprint(source, 'vm.dns_name', fact,
                                       'vm_dns_name', fingerprint)
         self._add_fact_to_fingerprint(source, 'vm.host.name',
```
Comment (myersCody) [inline quipucords/fingerprinter/task.py]: Just out of curiosity why do we still have the `vm.host.name` in the fingerprinter here?

File: quipucords/fingerprinter/tests_fingerprint_task.py [hunk 2]
```diff
@@ -410,7 +410,7 @@ def _validate_vcenter_result(self, fingerprint, fact):
         self.assertEqual(fact.get('vm.dns_name'),
                          fingerprint.get('vm_dns_name'))
         self.assertEqual(fact.get('vm.host.name'),
-                         fingerprint.get('vm_host'))
+                         fingerprint.get('virtual_host_name'))
 
         self.assertEqual(fact.get('vm.host.cpu_count'),
```
Comment (myersCody) [inline quipucords/fingerprinter/tests_fingerprint_task.py]: Do we need to test for the `virtual_host_name` fact here?  Am I right in thinking that the `vm.host.name` & `virtual_host_name` are separate facts?

File: quipucords/fingerprinter/task.py (hunk 1 above)
Comment (kholdaway) [reply quipucords/fingerprinter/task.py]: It's a mapping between details report names and deployment report names.

File: quipucords/fingerprinter/tests_fingerprint_task.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/fingerprinter/tests_fingerprint_task.py]: The names in details don't always match the deployments.

**Pull request merged**